notebooks/performance_metrics.ipynb

```diff
@@ -49,21 +49,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Use case: compute axon and myelin validation metrics on a sample image"
+    "## Use case: compute axon and myelin validation metrics on a sample image"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 32,
    "metadata": {},
    "outputs": [],
    "source": [
-    "path_img = '~/tmp/axondeepseg/AxonDeepSeg/data_test/image.png'"
+    "path_img = '../AxonDeepSeg/data_test/image.png'"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 33,
```

Comment:
_v1 is still in model_name on line 81 below

Reply:
resolved in commit(5afb9bb)